### ISSUE TITLE
Import error on Python 3.11

### ISSUE THREAD

**Issue by BonkaBonka** [Charahiro-tan/pal_rcon#1]
~~~
>>> from pal_rcon import PalRcon
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jamesc/build/a2s/lib/python3.11/site-packages/pal_rcon/__init__.py", line 1, in <module>
    from .async_pal_rcon import AsyncPalRcon
  File "/home/jamesc/build/a2s/lib/python3.11/site-packages/pal_rcon/async_pal_rcon.py", line 66
    + f"message={command.message if command.message != self.password else "**PW**"}",
                                                                           ^^
SyntaxError: f-string: expecting '}'
~~~

I was able to fix the issue by changing the double-quotes around `**PW**` to single quotes in `async_pal_rcon.py` just like you have in `pal_rcon.py`.

**Comment by BonkaBonka** [comment]
I should note that I installed the library into an existing venv using the following command:
~~~
~/build/a2s$ bin/pip install pal_rcon
Collecting pal_rcon
  Obtaining dependency information for pal_rcon from https://files.pythonhosted.org/packages/74/04/1d0147734b724dc98498d7e43cffa90e19eaaa0a84d89b0007fead32c97d/pal_rcon-0.0.2-py3-none-any.whl.metadata
  Downloading pal_rcon-0.0.2-py3-none-any.whl.metadata (4.7 kB)
Downloading pal_rcon-0.0.2-py3-none-any.whl (10 kB)
Installing collected packages: pal_rcon
Successfully installed pal_rcon-0.0.2
~~~

**Comment by Charahiro** [comment]
Hello, @BonkaBonka 
Indeed, there was a syntax error.
I have corrected it now, so it should be available as version 0.0.3.

**Comment by BonkaBonka** [comment]
Thank you!
